ark/utils/notebooks_test_utils.py

```diff
@@ -462,11 +462,11 @@ def flowsom_cell_setup(tb, flowsom_dir, pixel_dir, pixel_cluster_col='pixel_meta
     tb.execute_cell('import')
 
     # create data which will be loaded into img_xr -- ONLY FOR MANTIS PROJECT CREATION
-    tiff_dir = os.path.join(flowsom_dir, "input_data")
-    os.mkdir(tiff_dir)
+    tiff_dir = os.path.join(flowsom_dir, "image_data")
+    os.makedirs(tiff_dir)
 
     # create sample pixel output dir
-    os.mkdir(os.path.join(flowsom_dir, pixel_dir))
+    os.makedirs(os.path.join(flowsom_dir, "segmentation", pixel_dir))
```

Comment:
Didn't catch this in the first round, looks like a couple different paths will need to be changed. But I think this will be more intuitive. 

One top-level folder for the raw image data, one top-level folder for the segmentation data, and one top-level directory for the Pixie analyses

Reply:
Would it be 
1. `example_dataset/pixie/example_pixel_output_dir/` or 
2. `example_dataset/example_pixel_output_dir`?